Italian/finestra/tour/tour.md

```diff
@@ -1,4 +1,8 @@
 Con le tue tecniche di manipolazione mentale e la bambola voodoo riesci a domare il T-Rex, lo fai venire verso di te
 
 Ci salti su e vai alla scoperta di questo mondo fiabesco in cui sei capitato
+
+[Ti sembra solo un sogno](../../dormire/sogno-strano.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.